source/contracts/reporting/InitialReporter.sol

```diff
@@ -85,6 +85,12 @@ contract InitialReporter is DelegationTarget, Ownable, BaseReportingParticipant,
         return true;
     }
 
+    function forkAndRedeem() public onlyInGoodTimes returns (bool) {
```

Comment:
Because evidently that causes issue with pyethereum during testing :/ It wasn't getting added to the recognized functions for the ABIContract object so rather than debugging that this seemed more prrudent

Reply:
Ok, that seems like something that should be looked into eventually, but I agree this doesn't seem like it's worth spending much time on before MVP.